packages/alpha/src/types.ts

```diff
@@ -52,7 +61,7 @@ export interface SimulatorStepField {
   name: string;
   label: string;
   info: string;
-  options?: SimulatorStepFieldOption[];
+  options?: SimulatorStepFieldOption[] | null;
```

Comment:
```suggestion
  options?: SimulatorStepFieldOption[];
```
it's already optional